### PR TITLE
DashboardScene: Adjust plugin-e2e for dashboardScene FF

### LINKED ISSUE #89051: adjust playwrite tests that break with `dashboardScene` FF

### PR DESCRIPTION
**Pull request by Sergej-Vlasov**
Fixes #89051

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/dashboard-scene/panel-edit/PanelOptions.test.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ describe('PanelOptions', () => {
```
Comment (sunker) [inline public/app/features/dashboard-scene/panel-edit/PanelOptions.test.tsx:116]: Changes in this and the next file seem unrelated?

Comment (Sergej-Vlasov) [reply public/app/features/dashboard-scene/panel-edit/PanelOptions.test.tsx:116]: Hey @sunker! Option pane Title field selector was changed so in `e2e/plugin-e2e/plugin-e2e-api-tests/as-admin-user/panelEditPage.spec.ts` `edit panel plugin settings` broke. Adjusted selector to the one test expects 👍

**Review by sunker — APPROVED** [review]
LGTM

**Review by ivanortegaalba — APPROVED** [review]
